### PR TITLE
fix(icon): fix missing watcher for scale

### PR DESCRIPTION
**Pull request by jcfranco**
This adds a missing watcher for scale after `size` got dropped.

### CODE REVIEW

File: src/components/calcite-icon/calcite-icon.tsx [hunk 1]
```diff
@@ -160,7 +160,6 @@ export class CalciteIcon {
 
   @Watch("icon")
   @Watch("filled")
-  @Watch("size")
```
Comment (paulcpederson) [inline src/components/calcite-icon/calcite-icon.tsx]: don't you need to change this to `scale` rather than removing?

Comment (jcfranco) [reply src/components/calcite-icon/calcite-icon.tsx]: D-oh! Yes, good catch. I'll get that fixed right away.

Comment (jcfranco) [reply src/components/calcite-icon/calcite-icon.tsx]: 💥

**Comment by macandcheese** [comment]
🚀

File: src/components/calcite-icon/calcite-icon.tsx (hunk 1 above)
Comment (jcfranco) [reply src/components/calcite-icon/calcite-icon.tsx]: 💥

**Pull request merged**